### DISCUSSION TITLE
寻狗启事

### DISCUSSION THREAD

**Discussion by Shadow00624** [share121/inter-knot#5326] [委托]
如题，在贝洛伯格遛狗时，狗绳不慎松动，致使狗狗调皮逃脱。听说绳匠找东西很厉害，特来绳网委托。
现已报警，银鬃铁卫帮我在大街小巷张贴了如图所示的寻狗启事。
介于我家狗狗有乔装技能和诈骗前科，未成年的、喜欢大波浪漂亮姐姐的和手机上没下载反诈中心的，见到图片上的人请立刻报警。
如有线索，烦请前往星穹列车（当前坐标：罗浮仙舟）寻找 星 女士，必有重谢。
电话：54188-54188
邮件：Master_of_Lordly_Trashcan@YHQBX.com
（在提瓦特当太多次狗了，好不容易有一条自己的狗狗，虽然以为是波桑姐姐被网骗了，但毕竟是珍贵的第一次啊！救救孩子吧！飙泪.jpg）
![1727066573374e2c9200f51c1c01ffcc773c1f815d68651fcdcbce34d2ab874957591126d5464 0](https://github.com/user-attachments/assets/467f4b55-7d8a-4a62-8656-8f742cb1af61)
![1727066771248c245c703929e295d4f83f7235e94e1e6843f8a3ffe8656f25197b88c9fd19051 0](https://github.com/user-attachments/assets/5067d8c8-7c76-42dd-b1e1-87fcfdc5ce9e)

**Comment by Oumainory** [comment]
这个委托，我接了！<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by aomengxin** [comment]
可恶，委托没了
